internal/command/commands/service.go

```diff
@@ -42,16 +41,12 @@ func (s *Service) Run(args []string) int {
 
 	if s.Create {
 
-		if *rebuild {
-			emptyParameters := emptyParameters(map[string]string{"--name": *serviceName, "--version": *serviceVersion})
-			if len(emptyParameters) == 0 {
-				serviceClient.RebuildService(*serviceName, *serviceVersion)
-				s.Logger.Output("Command to check status of images")
-				s.Logger.ItalicEmphasize(fmt.Sprintf("odin status service --name %s --version %s", *serviceName, *serviceVersion))
-				return 0
-			}
-			s.Logger.Error(fmt.Sprintf("%s cannot be blank", emptyParameters))
-			return 1
+		emptyParameters := emptyParameters(map[string]string{"--name": *serviceName, "--version": *serviceVersion})
+		if len(emptyParameters) == 0 {
+			serviceClient.RebuildServiceStream(*serviceName, *serviceVersion)
```

Comment:
should `RebuildServiceStream` be executed every time?
What if first time service create?